src/account/operations/syncing/mod.rs

```diff
@@ -43,7 +43,7 @@ impl AccountHandle {
             .as_millis();
         let mut last_synced = self.last_synced.lock().await;
         log::debug!("[SYNC] last time synced before {}ms", time_now - *last_synced);
-        if time_now - *last_synced < MIN_SYNC_INTERVAL && !options.force_syncing {
+        if !options.force_syncing && time_now - *last_synced < MIN_SYNC_INTERVAL   {
```

Comment:
How is this an improvement?
Not sure what exactly the compiler creates out of it, but to me it looks like it would rather be an improvement if you check for `!options.force_syncing` first, because then it doesn't has to do the calculation of the time

Reply:
If you compare two booleans with a && usually if the first boolean is wrong the second one wont be looked at for the first place because the whole expression is already false.
With this you "save" the calculation after that if you have the boolean true before &&

Reply:
If it bothers you I can removed it

Reply:
Sorry, it's alright this way :+1:  I commented on my phone and there it looked like there was a second if statement before from which you moved it to a later point